t/e2e/whisper/mailservice_test.go

```diff
@@ -31,10 +31,11 @@ func (s *MailServiceSuite) TestShhextRequestMessagesRPCMethodAvailability() {
 
 	s.StartTestNode(func(config *params.NodeConfig) {
 		config.RPCEnabled = true
+		config.APIModules = "shhext"
 	})
 	defer s.StopTestNode()
 
-	client := s.StatusNode.RPCClient()
+	client := s.StatusNode.RPCPrivateClient()
```

Comment:
If `shhext` is set in `APIModules` list, then it should work fine here with `RPCClient`, right?